features/02_contact.feature

```diff
@@ -0,0 +1,11 @@
+Feature: Send a Message through the contact button
+
+ As a common user
+    I want to use the contact button
+    In order to send a message
+
+Scenario: Sending messages through the contact button
+        Given the user is on contact button
+        When the user sends a message
+        Then the page confirm message is displayed
```

Comment:
Then the page displays a confirmation message

that sounds better (this is just English syntax so I'm not going hard on these corrections, just know that there are more clear ways of writing